app/main/views/briefs.py

```diff
@@ -208,7 +208,12 @@ def _render_not_eligible_for_brief_error_page(brief, clarification_question=Fals
     has_framework_lot_service = has_framework_service and bool(data_api_client.find_services(
```

Comment:
I know it's not technically to do with this story, but it looks to me like this check which calls `data_api_client.find_services()` twice could be more efficient.  Seems like we could just get the services once and then see if any of them are in the right lot.